graphing/src/cloudevolution/views.py

```diff
@@ -94,19 +95,71 @@ def expt_name(request, experiment):
 
 	return render(request, "experiment.html", context)
 
+
 def dilutions(request, experiment):
 	sidebar_links, subdir_log = file_scan('expt')
 	vial_count = range(0, 16)
 	expt_dir, expt_subdir = file_scan(experiment)
 	rootdir = os.path.dirname(os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__)))))
 	evolver_dir = os.path.join(rootdir, 'experiment')
 	pump_cal = os.path.join(evolver_dir, expt_subdir[0], "pump_cal.txt")
+	bottle_file = os.path.join(evolver_dir, expt_subdir[0], "bottles.txt")
+	expt_pickle = os.path.join(evolver_dir, expt_subdir[0], expt_dir[0], expt_dir[0] + ".pickle")
+
+	if not os.path.isfile(bottle_file):
+		open(bottle_file, 'w')
```

Comment:
File could be created in `eVOLVER.py` when the rest of the experiment files are created, but I did not want to touch It for the moment.

Reply:
It's better to just have the script generate a generic file if it is not created by the user. Or have a generic file pre-checked in to the template directory and add some documentation around the usage of it.